layouts/partials/css.html

```diff
@@ -13,6 +13,14 @@
 
 <link rel="icon" href="{{ "images/favicon.ico" | relURL }}" />
 <link rel="stylesheet" href="{{ $fontUrl }}">
+
+<!-- Fallback font for symbols (such as "🛈"). -->
```

Comment:
Why don't we just add this as a static entry, instead of making it configurable?

I also don't think we need 300, 400, 600, and 700 just for the symbol.

Reply:
Changed in https://github.com/scientific-python/scientific-python-hugo-theme/pull/268/commits/2e0689fb160a56b7e6dc1a86d65d2de63e116c38.  Thanks.